docs/interactions/web-messaging/source.md

```diff
@@ -105,12 +105,24 @@ This parameter is mainly used in the mobile single-threaded part of this connect
 !!! warning
     This parameter is also used for auto closing the messaging threads. The messaging thread will be automatically closed by the application if the inactivity in the thread is at least the max between this parameter and 4 days. This is to ensure we do not leave open threads forever.
 
+### Enable Outbound Messaging
+
+This setting is available for web messaging, iOS messaging and Android messaging. It allows to initiate discussions on behalf of an agent through Engage Digital's API.
+
+It can be configured under the "Advanced settings" section:
+
+<img class="img-fluid" width="276" src="../../../img/web-messaging-enable-outbound-messaging.png">
+
+Enabling this option will unlock the `Initiate discussion` permission for this source. You will then be able to add it to agents you want to send messages on behalf of.
+
```

Comment:
I added doc for the channel settings but maybe it's too early, WDYT? @DrissTM

Reply:
why do you think it's too early? won't they need to configure this in order to send outbound messages?

Reply:
Because the checkbox is techteam only

Reply:
let's still add this documentation as it won't make sense to have the api part without the settings to enable it